lib/open_api/serializer.rb

```diff
@@ -11,6 +11,7 @@ def as_json(arg = {})
       schema["properties"].keys.each do |name|
         attrs[name] = attrs[name].iso8601 if attrs[name].kind_of?(Time)
         attrs[name] = attrs[name].to_s if name.ends_with?("_id") || name == "id"
+        attrs[name] = self.public_send(name) unless attrs.key?(name)
```

Comment:
looks like this would clash with encryption_filtered

Reply:
@bdunne btw. why are we turning id attributes into strings? Those should remain integers? Also in the swagger yaml, we are saying these are format: uuid, but those are integers.